library/swift/test/HttpBridgeTests/ReceiveDataTest.swift

```diff
@@ -0,0 +1,78 @@
+import Envoy
+import EnvoyEngine
+import Foundation
+import XCTest
+
+final class ReceiveDataTests: XCTestCase {
+  func testReceiveData() throws {
+    // swiftlint:disable:next line_length
+    let apiListenerType = "type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager"
+    // swiftlint:disable:next line_length
```

Comment:
If you want you could put one of these at the top of the individual file and do `// swiftlint:disable line_length`

Reply:
just want to do these specific lines